lib/cfonb/operation_detail/mmo.rb

```diff
@@ -14,6 +14,8 @@ def self.apply(operation, line)
         sign = operation.amount <=> 0 # the detail amount is unsigned
 
         operation.original_amount = sign * BigDecimal(line.detail[4..17]) / (10**scale)
+        exchange_rate_value = line.detail[26..29]
+        operation.exchange_rate = BigDecimal(exchange_rate_value) / 1000 if exchange_rate_value
```

Comment:
we are missing the scale of the exchange rate also
```suggestion
        exchange_rate_value = line.detail[26..29]
        exchange_rate_scale = line.detail[18]
        operation.exchange_rate = BigDecimal(exchange_rate_value) / (10**exchange_rate_scale) if exchange_rate_value
```

Reply:
@frantisekrokusek make sense. thank
code is updated